app/src/main/java/com/sun/moviedb/data/repository/MovieByCategoryRepository.kt

```diff
@@ -0,0 +1,47 @@
+package com.sun.moviedb.data.repository
+
+import androidx.lifecycle.LiveData
+import com.sun.moviedb.base.BaseResponse
+import com.sun.moviedb.base.ContextProviders
+import com.sun.moviedb.base.NetworkBoundResource
+import com.sun.moviedb.data.DatabaseManager
+import com.sun.moviedb.data.dto.CategoryDto
+import com.sun.moviedb.data.dto.MovieDto
+import com.sun.moviedb.data.entity.Movie
+import com.sun.moviedb.data.remote.RemoteDataSource
+import com.sun.moviedb.utils.CategoryQuery
+
+/**
+ * Created by nguyenxuanhoi on 2019-08-20.
+ * @author nguyen.xuan.hoi@sun-asterisk.com
+ */
+class MovieByCategoryRepository(val coroutines: ContextProviders, private val remoteDataSource: RemoteDataSource) :
+    MovieByCategoryContract {
+
+    override fun getQueryTypeCategory(): List<CategoryDto> = listOf(
```

Comment:
```suggestion
    override fun getQueryTypeCategory() = listOf(
```
kotlin có trình biên dịch kiểu mạnh
em có thể viết ntn